server/setup_postgresql_local_dev.sh

```diff
@@ -1,4 +1,4 @@
 sudo -u postgres psql -p $PG_PORT -c "CREATE ROLE $PG_USER PASSWORD '$PG_PASSWORD' LOGIN"
 sudo -u postgres createdb -p $PG_PORT -O $PG_USER $PG_USER
 sudo -u postgres psql -p $PG_PORT -d $PG_USER -c 'CREATE EXTENSION IF NOT EXISTS "pgcrypto"'
-./setup_postgresql.sh 'MyClient' '+12223334444' '+19998887777'
\ No newline at end of file
+./setup_postgresql.sh $PG_PASSWORD $PG_USER $PG_HOST $PG_PORT $PG_USER
```

Comment:
Thanks for fixing this! I guess it wasn't updated when `setup_postgresql.sh` was changed to match Sensors.

Reply:
I guess so! And I believe it worked as setup_postgresql.sh defaults to using the environment variables if they exist, anyways.